dev/com.ibm.ws.app.manager.wab/src/com/ibm/ws/app/manager/wab/internal/WABInstaller.java

```diff
@@ -459,6 +466,23 @@ public boolean getUseJandex() {
                         DeployedModuleInfo deployedMod = deployedApp.getDeployedModule(moduleInfo);
                         wab.setDeployedModuleInfo(deployedMod);
 
+                        WebAppConfiguration appConfig = (WebAppConfiguration)((WebModuleMetaData)mmd).getConfiguration();
```

Comment:
This doesn't handle when configuration of the variable changes.  But I think you know that.  Do we intend to handle that in the future?

Reply:
Right, I've just updated the PR to deal with that. The PR from this morning was incomplete, I wanted to get a build running so Navid could run some testing. I should have commented in the PR on that part.